### ISSUE TITLE
Error in Loading the Feature extractor 

### ISSUE THREAD

**Issue by guneetsk99** [huggingface/transformers#17244]
### System Info

```shell
WARNING:tensorflow:From /usr/local/lib/python3.7/dist-packages/transformers/commands/env.py:52: is_gpu_available (from tensorflow.python.framework.test_util) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.config.list_physical_devices('GPU')` instead.
2022-05-13 16:11:52.801587: W tensorflow/core/common_runtime/gpu/gpu_bfc_allocator.cc:39] Overriding allow_growth setting because the TF_FORCE_GPU_ALLOW_GROWTH environment variable is set. Original config value was 0.

Copy-and-paste the text below in your GitHub issue and FILL OUT the two last points.

- `transformers` version: 4.18.0
- Platform: Linux-5.4.188+-x86_64-with-Ubuntu-18.04-bionic
- Python version: 3.7.13
- Huggingface_hub version: 0.6.0
- PyTorch version (GPU?): 1.11.0+cu113 (True)
- Tensorflow version (GPU?): 2.8.0 (True)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: Yes
```

### Who can help?

@sgugger @LysandreJik 
![image](https://user-images.githubusercontent.com/43180442/168324537-ed04b1e7-f47b-41f1-8213-11759c7cb1b1.png)
I am facing this error 
I am not able to figure out how to get past it. Till yesterday I was running the same code and didnt face any errors but today when I ran it to reproduce the results this happened can you please help 

### Information

- [ ] The official example scripts
- [X] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [X] My own task or dataset (give details below)

### Reproduction

```

from transformers import ConvNextFeatureExtractor, ConvNextForImageClassification
from hugsvision.nnet.VisionClassifierTrainer import VisionClassifierTrainer
from transformers import AutoFeatureExtractor, SwinForImageClassification
from transformers import ViTFeatureExtractor, ViTForImageClassification
from transformers import BeitFeatureExtractor, BeitForImageClassification
from transformers import PerceiverFeatureExtractor, PerceiverForImageClassificationLearned
trainer = VisionClassifierTrainer(
	model_name   = "ViT-Model",
	train        = train,
	test         = test,
	output_dir   = "./out/",
	max_epochs   = 5,
	batch_size   = 16, # On RTX 2080 Ti
	lr	     = 0.0003,
	fp16	     = True,
	model = ConvNextForImageClassification.from_pretrained(
	    huggingface_model,
	    num_labels = 5,
	    label2id   = label2id,
	    id2label   = id2label,
		use_auth_token=True,
		ignore_mismatched_sizes=True
	),
	feature_extractor = ConvNextFeatureExtractor.from_pretrained(
		huggingface_model,
	),
)
```

### Expected behavior

```shell
I want to know why this error arose which till yesterday didnt even exist.
Just to make a note Hugsvision didnt update their code base in past one day hence I reached out to you all for help 
Thanks in advacne
```

**Comment by sgugger** [comment]
This is fixed by #17239
Will make a patch for PyPi.

**Comment by LysandreJik** [comment]
Patched on pypi!
